# Requests that an end effector writes data to an RFID tag

# True if data should be written from the bottom foot, otherwise it should write from the top
bool bottom_foot

# Block number to write on RFID tag. Should be >=4, since 0-3 are manufacturer data
uint8 block

uint8[] data

---

bool success